e2e/src/recursive_search.spec.ts

```diff
@@ -23,18 +25,16 @@ suite('Recursive search', () => {
   });
 
   test('Should unregister deleted columns', async () => {
-    await activateAndWait(TABLE_DOES_NOT_EXIST_DOC_URI);
-    await replaceText(', 2 as amount', '');
-
-    await activateAndWait(RECURSIVE_SEARCH_DOC_URI);
-    await replaceText('s.amount', 's.amount + 2');
+    await activateAndWait(CURRENT_TIME_DOC_URI);
```

Comment:
Should we call `activateAndWait(CURRENT_TIME_OF_DAY_DOC_URI);` first to ensure that we filled catalog with CURRENT_TIME_DOC_URI?

Reply:
Seems not. After opening of `current_time.sql` all its columns will be registered in catalog.
Here we check that after removal of `hour` column it will be removed from catalog. And it will cause an error in dependent model `current_time_of_day.sql`.

Reply:
Ok, sounds good.